docs/notebooks/4.1 Setting up and Using PV Sites.ipynb

```diff
@@ -0,0 +1,521 @@
+    "\n",
+    "from solcast import live, forecast, pv_power_sites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c07c2f06-fea6-4af2-881f-4a2b5ebfbb96",
+   "metadata": {},
+   "source": [
+    "## Get Data at an Existing Site\n",
+    "\n",
+    "You can list the PV Power Sites available to you.  These include test sites at Solcast's [unmetered locations](https://docs.solcast.com.au/#unmetered-locations), as well as any sites you have set up for yourself.  For instance, you can list the available PV Power Sites and find the test site at the Giza Pyramid Complex."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "c17bdd85-0269-4015-9e27-019066887672",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'resource_id': '8d10-f530-af85-5cbb',\n",
+       "  'name': 'Test Site: Giza Pyramid Complex',\n",
+       "  'latitude': 29.977296,\n",
+       "  'longitude': 31.132496,\n",
+       "  'capacity': 145,\n",
+       "  'capacity_dc': 180,\n",
+       "  'azimuth': 180,\n",
+       "  'tilt': 27.28,\n",
+       "  'tracking_type': 'horizontal_single_axis',\n",
+       "  'install_date': '2021-07-01T00:00:00.0000000Z',\n",
+       "  'module_type': 'poly-si',\n",
+       "  'ground_coverage_ratio': 0.36,\n",
+       "  'derating_temp_module': 0.0039,\n",
+       "  'derating_age_system': 0.0059,\n",
+       "  'derating_other_system': 0.0221,\n",
+       "  'inverter_peak_efficiency': 0.985,\n",
+       "  'tracker_axis_azimuth': 0,\n",
+       "  'tracker_max_rotation_angle': 60,\n",
+       "  'tracker_back_tracking': True,\n",
+       "  'tracker_smart_tracking': False,\n",
+       "  'terrain_slope': 0,\n",
+       "  'terrain_azimuth': 0,\n",
+       "  'dust_soiling_average': [0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015,\n",
+       "   0.015],\n",
+       "  'bifacial_system': False,\n",
+       "  'site_ground_albedo': 0.25,\n",
+       "  'bifaciality_factor': 0.7,\n",
+       "  'pvrow_height': 1.5,\n",
+       "  'pvrow_width': 2,\n",
+       "  'is_unmetered': True,\n",
+       "  'confirmed_metadata': '2022-11-04T03:13:07.6874000Z'}]"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "res = pv_power_sites.list_pv_power_sites()\n",
+    "[s for s in res.to_dict() if s[\"name\"] == 'Test Site: Giza Pyramid Complex']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba589fe6-43df-4efb-b7ba-441e0120180c",
+   "metadata": {},
+   "source": [
+    "Note that the site configuration has a unique `resource_id`, in this case '8d10-f530-af85-5cbb'. We need to use this to access data modelled using this site's configuration, which is for a 145 kW capacity tracking array."
```

Comment:
Expand this to explain explicitly that PV Power sites are all identified by a unique resource_id e.g.

"Solcast Advance PV Power sites are identified by a unique ID, referred to as a `resource_id`. This can found in the site configuration, in this case '8d10-f530-af85-5cbb'. The `resource_id` is used request and update the site's configuration and to fetch PV generation estimates, as shown below."

Reply:
done